BlackJackGame/Deck.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace Black_Jack_Game
 {
-    public class Deck
+    public class Deck : IDeck
```

Comment:
Ok soo this whole Deck class is kinda weird. You've kind of written each method as if they were meant to be static methods. I'll explain why for each method